sktime/utils/validation/__init__.py

```diff
@@ -38,12 +43,33 @@ def check_n_jobs(n_jobs):
         return n_jobs
 
 
-def check_window_length(window_length, name="window_length"):
+def check_window_length(window_length, n_timepoints=None, name="window_length"):
     """Validate window length"""
+    """
+    Parameters
+    ----------
+    window_length: positive int or positive float
+        The number of training set used for splitting
+        Window length for transformed feature variables
+
+    Returns
+    -------
+    window_length: int
+    """
     if window_length is not None:
-        if not is_int(window_length) or window_length < 1:
+
+        if n_timepoints is None:
```

Comment:
Why do we set n_timepoints to 1 here? I think perhaps we should:
* ignore n_timepoints if window_length is a positive integer
* raise an error if n_timepoints is None if window_length is a positive float (instead of setting it to 1)

Reply:
Noted